frontend/components/Header/_index.html.erb

```diff
@@ -0,0 +1,3 @@
+<header class="Header">
+  <%= image_tag asset_pack_path('components/Header/static/logo.svg'), class: 'Header-logo', alt: 'Barnardo’s, Believe in children' %>
```

Comment:
Should this text be in the `en.yml`?

Reply:
It should be, there is a lot of text that should be moved to `en.yml`. The [current alt text](https://github.com/barnardos/consent-form-builder/blob/master/app/views/layouts/application.html.erb#L29) is also hardcoded in the view.